app/controllers/occupations_controller.rb

```diff
@@ -0,0 +1,22 @@
+class OccupationsController < ApplicationController
+  def update
+    report = Report.find_for_guid(params[:report_id])
+    occupation = report.find_occupation(params[:id].to_i)
+    if occupation_accepted?(params)
+      occupation.mark_as_accepted_with_actions(action_params)
+    else
+      occupation.mark_as_refused
+    end
+    redirect_to report_path(report)
+  end
+
+  private
+
+  def occupation_accepted?(params)
+    params[:commit].downcase != 'refuse'
+  end
+
+  def action_params
+     params.select {|key, value| ["actions", "notes"].include?(key) }
```

Comment:
Rails 4 has [built in support](http://guides.rubyonrails.org/action_controller_overview.html#strong-parameters) for this, like so:
```ruby
params.permit(:actions, :notes)
```
or, if they're required params (this blows up if any required params are missing though):
```ruby
params.require(:actions, :notes)
```